pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -463,6 +465,7 @@ stdenv.mkDerivation (finalAttrs: {
     (enableFeature withGme "libgme")
     (enableFeature withGnutls "gnutls")
     (enableFeature withGsm "libgsm")
+    (enableFeature withHarfbuzz "libharfbuzz")
```

this breaks compiling on versions older than 6.1

```suggestion
  ] ++ optionals (versionAtLeast finalAttrs.version "6.1") [
    (enableFeature withHarfbuzz "libharfbuzz")
  ] ++ [
```